src/irmin-pack/unix/file_manager.ml

```diff
@@ -231,18 +230,17 @@ struct
     t.mapping <- mapping;
     Ok ()
 
-  let reopen_suffix t ~generation ~end_poff =
+  let reopen_suffix t ~generation ~end_poff ~chunk_num =
```

Comment:
Heads up that in my prototyping for GC I noticed that you need to pass `~chunk_idx` here so I already added it to #2117 so it'll be picked up next time you rebase. Sorry for the conflict. :see_no_evil:

Reply:
no problem, its normal to have conflicts :)